lib/knock/jwt_adapter.rb

```diff
@@ -0,0 +1,23 @@
+module Knock
+  class JwtAdapter
+    def decode(token, options)
+      payload, _ = JWT.decode token.to_s, decode_key, true, options
```

Comment:
Not using parens for `decode` and `encode` makes it hard to see where the arguments begin and end.